frontend/src/ui/blocks/Text.tsx

```diff
@@ -34,8 +34,8 @@ const MARKDOWN_COMPONENTS: TransformOptions["components"] = {
         css={{
             borderLeft: "4px solid #e5e5e5",
             padding: "2px 8px",
-            "& > *:first-child": { marginTop: 0 },
-            "& > *:last-child": { marginBottom: 0 },
+            "& > *:first-of-type": { marginTop: 0 },
```

Comment:
Ah it's terrible. It has to do with emotion-js. If it is used for SSR, they do inserted a `<style>` tag before every element that is styled. This would then mess up `:first-child` logic, right? Thus the library warns about it. There are long GitHub threads were people argue the library should not warn about it or there should be at least a simple way to disable it. Not everyone uses SSR after all. So far, there doesn't seem a way to disable the warning. Which is super annoying because our plyr player uses `:first-child` too, so I see the warning every time I visit the video page. It's... annoying.

Reply:
Oooh, interesting. :eyes: Should we have an issue to track this?

Reply:
Yeah maybe. Would like to just disable the warning somehow :/

Reply:
Ok so I silenced the warning now, actually. The commit is in this PR: https://github.com/elan-ev/tobira/pull/227 ("Silence emotion-js warnings about :first-child").